middleware/webhooks.js

```diff
@@ -44,21 +46,20 @@ function parsePolicyViolation(entry) {
     page_id: entry[0].id
   };
 }
-
+// http://m.me/109461977131004?ref=command=start_book,book_id=1
```

Comment:
I hadn't considered adding a referral link example in the code but I think it's a really good idea, but it might make more sense for it to be `event.referral` section.  Do you want to move it there and add a comment before that says "Referral link sample:" or something?